stdlib/public/Cxx/CxxConvertibleToCollection.swift

```diff
@@ -38,13 +38,13 @@ internal func forEachElement<C: CxxConvertibleToCollection>(
 }
 
 extension Array {
-  /// Creates an array containing the elements of a C++ collection.
+  /// Creates an array containing the elements of a C++ container.
   ///
-  /// This initializer copies each element of the C++ collection to a new Swift
+  /// This initializer copies each element of the C++ container to a new Swift
   /// array.
   ///
   /// - Complexity: O(*n*), where *n* is the number of elements in the C++
-  ///   collection.
+  ///   container.
```

Comment:
Wondering if we should add a note (for all these APIs) saying "Note that copying each element of the array may not take constant time, if the type of the element is defined in C++".

Reply:
That's reasonable, I added a note about that, thanks.